### PR TITLE
Fix Class does not comply with psr-4 autoloading standard

### PR DESCRIPTION
**Pull request by Jimbolino**
fixes: https://github.com/SocialiteProviders/Providers/issues/402
I've only renamed Folders, the split repositories should be the same.
So people relying on those should be safe.

- [ ] Release major deviantart version
- [ ] Release major mattermost version
- [ ] Release 23andme
- [ ] Release 37Signals
- [ ] Release 500px
- [ ] Release App.net
- [ ] Release Battle.net
- [ ] Release Bit.ly
- [ ] Release Human-API
- [ ] Release Mediacube
- [ ] Release Microsoft-Azure
- [ ] Release Microsoft-Graph
- [ ] Release Microsoft-Live
- [ ] Release Microsoft-TeamService
- [ ] Release PayPal-Sandbox
- [ ] Release Weixin-Web

### CODE REVIEW

File: src/Deviantart/Provider.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace SocialiteProviders\deviantART;
+namespace SocialiteProviders\Deviantart;
```
Comment (owenvoke) [inline src/Deviantart/Provider.php:3]: Should the DeviantArt classes be under a `DeviantArt` directory and namespace to match the site naming?
```suggestion
namespace SocialiteProviders\DeviantArt;
```

Comment (Jimbolino) [reply src/Deviantart/Provider.php:3]: I'm matching the namespace as defined in DeviantartExtendSocialite.php
Changing the namespace of that file, would probably break a couple of people who already use it: https://packagist.org/packages/socialiteproviders/deviantart

Comment (owenvoke) [reply src/Deviantart/Provider.php:3]: Ah, ok. 👍

**Comment by Jimbolino** [comment]
all better now :)

File: src/Deviantart/Provider.php (hunk 1 above)
Comment (atymic) [reply src/Deviantart/Provider.php:3]: Thanks. Let me check if there's any other major version bumps required.

File: src/Mattermost/MattermostExtendSocialite.php [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 use SocialiteProviders\Manager\SocialiteWasCalled;
 
-class MatterMostExtendSocialite
+class MattermostExtendSocialite
```
Comment (atymic) [inline src/Mattermost/MattermostExtendSocialite.php:7]: Guess this will need a major version bump as well?

Comment (Jimbolino) [reply src/Mattermost/MattermostExtendSocialite.php:7]: nope, it was actually the "style fixes" commit that changed it, i'm simply changing it back:
https://github.com/SocialiteProviders/Providers/commits/master/src/Mattermost/MatterMostExtendSocialite.php

Comment (atymic) [reply src/Mattermost/MattermostExtendSocialite.php:7]: Ahh. good catch, I will check if any other ones were changed.

File: src/Deviantart/Provider.php (hunk 1 above)
Comment (owenvoke) [reply src/Deviantart/Provider.php:3]: If you're doing a major version for DeviantArt, is it worth making it consistent with the actual site naming then? 🤨

Comment (atymic) [reply src/Deviantart/Provider.php:3]: Yep, feel free :)

**Comment by atymic** [comment]
@Jimbolino did you want to fix deviant art still, or should I tag the new major version? :)

**Comment by owenvoke** [comment]
Oh, that was actually a request from me. I think it would make sense to fix it, however I can't at the moment.

**Comment by Jimbolino** [comment]
My preference is towards backwards compatibility.
So only change the name if it's really needed.
For example when a company changes its name, and new users might not recognize the old one.

**Comment by atymic** [comment]
Ok, no worries.
Will tag a major release now then :)

**Pull request merged**